minimalkv/net/boto3store.py

```diff
@@ -1,99 +1,15 @@
-import io
-from contextlib import contextmanager
-from shutil import copyfileobj
-from typing import List
+from minimalkv import CopyMixin, UrlMixin
+from minimalkv.fsspecstore import FSSpecStore
 
-from minimalkv import CopyMixin, KeyValueStore, UrlMixin
+try:
+    from s3fs import S3FileSystem
 
+    has_s3fs = True
+except ImportError:
+    has_s3fs = False
```

Comment:
how is this `has_s3fs` variable used?

Reply:
I forgot to use it like here:
https://github.com/data-engineering-collective/minimalkv/blob/be3e1b26c6b64eed3949180c9bbad9be93a7f410/minimalkv/net/gcstore.py#L56-L58